community/working-groups/supply-chain-security/suply-chain-security-paper-v2/SSCBPv2.md

```diff
@@ -0,0 +1,1270 @@
+# Software Supply Chain Best Practices v2
+
+Authors: Original White Paper authors, Marina Moore, Michael Lieberman, John Kjell, James Carnegie, Ben Cotton
+Reviewers:
+
+# Scope: {#scope:}
```

Comment:
It looks like GitHub auto-generates this heading ids: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#section-links.